app/app/src/main/java/com/example/clubwat/viewmodels/SignUpViewModel.kt

```diff
@@ -17,11 +17,25 @@ class SignUpViewModel(private val userRepository: UserRepository) : ViewModel()
     var password = mutableStateOf("")
     var passwordError = mutableStateOf<String?>(null)
     var allValuesError = mutableStateOf<String?>(null)
+    var emailError = mutableStateOf<String?>(null)
 
     fun createUser() {
         userRepository.createUser(firstName, lastName, email, password)
     }
 
+    fun isWaterlooEmail(email: String): Boolean {
```

Comment:
This is already done in the backend so I don't think we need the logic here. Also, I think @Kggupta addressed this in this PR?: https://github.com/Kggupta/cs446-project/pull/31/files